tests/AssertAnnotationTest.php

```diff
@@ -1241,7 +1241,7 @@ function test($a, $b): string {
                         return substr($a, 0, 1) . substr($b, 0, 1);
                     }'
             ],
-            'SKIPPED-convertConstStringType' => [
+            'convertConstStringType' => [
```

Comment:
I've re-enabled this test case as it is fixed with this PR as well.
Thanks to @orklah for referring to it in #6300